### ISSUE TITLE
Results on ETH3D

### ISSUE THREAD

**Issue by cnexah** [gangweiX/IGEV#37]
Hi, thank you for your great work!
I have one question about results on ETH3D:
What is the difference between the model sceneflow.pth and eth3d.pth? Did you fine-tune on training set of ETH3D?

**Comment by gangweiX** [comment]
Yeah, I fine-tune on training set and other similar dataset, refer to UniMatch (TPAMI 2023).

**Comment by cnexah** [comment]
I see. Thank you very much!
